tests/e2e/framework/kubernetes.go

```diff
@@ -43,6 +45,12 @@ var (
 	caHub      = flag.String("ca_hub", "", "Ca hub")
 	caTag      = flag.String("ca_tag", "", "Ca tag")
 	verbose    = flag.Bool("verbose", false, "Debug level noise from proxies")
+
+	modules = []string{
```

Comment:
I think we can keep using istio-install for now just following kubeTest.sh. We can move to istio-auth-16 later.